ui/app/templates/clients/index.hbs

```diff
@@ -3,14 +3,14 @@
     {{partial "partials/forbidden-message"}}
   {{else}}
     <div class="toolbar">
-      {{#if nodes.length}}
-        <div class="toolbar-item">
+      <div class="toolbar-item">
+        {{#if nodes.length}}
           {{search-box
             searchTerm=(mut searchTerm)
             onChange=(action resetPagination)
             placeholder="Search clients..."}}
-        </div>
-      {{/if}}
+        {{/if}}
+      </div>
```

Comment:
I’d prefer a pure CSS solution to this but I think that would require using grids.